docs/index.markdown

```diff
@@ -77,9 +77,17 @@ The EAR Production Suite is a joint [open-source development](https://github.com
 </div>
 
 <div markdown="1" class="text_section">
-## Installation
-The EPS is designed for REAPER 64-bit, on a 64-bit OS (macOS or Windows). The EAR Production Suite carries the <a href="https://github.com/ebu/ear-production-suite/blob/main/LICENSE.TXT">GPL v3.0 license</a>.
-<br /><br />
+## Download and Install
+Download the latest EAR Production Suite release (v0.7.0):
+  <div class="button-grid">
+    <a href="###########################!!!TODO!!!############################"><button class="c-btn">📦 Download Windows (x64)</button></a>
```

Comment:
Now only requires `###########################!!!TODO!!!############################` links replacing. 3 instances of.